invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -184,6 +184,17 @@ def _read_resource_type(self, id_):
             system_identity
         )._record
 
+    def _read_language(self, id_):
+        """Retrieve resource type record using service."""
+        lang = vocabulary_service.read(
+            ('languages', id_),
+            system_identity
+        )._record
+
+        alpha_2 = lang.get("props", {}).get("alpha_2", '')
+
+        return alpha_2 if alpha_2 else missing  # account for ''
```

Comment:
cosmetic: another option (`props` should be always there?):
```suggestion
        alpha_2 = lang["props"].get("alpha_2", None)
        return alpha_2 or missing
```

Reply:
> cosmetic: another option (props should be always there?):

Should be if people use our `language.py` script... We sort of need it for functionality, so I would say *yes*... but can make it optional too. WDYT?